README.md

```diff
@@ -643,340 +531,322 @@ terragrunt = {
         "refresh"
       ]
     }
+  }
+}
+```
 
-    extra_arguments "json_output" {
-      arguments = [
-        "-json"
-      ]
-      commands = [
-        "output"
-      ]
-    }
+With the configuration above, when you run `terragrunt apply`, Terragrunt will call Terraform as follows:
 
-    extra_arguments "fmt_diff" {
-      arguments = [
-        "-diff=true"
-      ]
-      commands = [
-        "fmt"
-      ]
-    }
+```
+> terragrunt apply
 
-  }
-}
+terraform apply -var bucket=example.bucket.name
 ```
 
-### The apply-all, destroy-all, and output-all commands
 
-Let's say you have a single environment (e.g. `stage` or `prod`) that has a number of Terraform modules within it:
+
+
+
+
+### Work with multiple Terraform modules
```

Comment:
Well, it also allows you to destroy and run output for multiple modules, so I'll stick with "work with".